app/assets/images/empty-profile-header.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
can we use font awesome for this? is it a different icon?

Reply:
I asked becca for the icon, looks like it is just the font awesome one. I've updated this.